roles/cifmw_external_dns/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure requirements are satisfied before beginning
+  ansible.builtin.include_tasks: requirements.yml
+
+- name: Ensure DNS entry is created
+  ansible.builtin.include_tasks: dns.yml
+
+- name: Create certificate if paths do not exist on target host
+  ansible.builtin.include_tasks: cert.yml
+  when: (not cert_stat_result.stat.exists | bool) or
```

Comment:
Note for reviewers: `cert_stat_result` and `key_stat_result` are created in the requirement.yml

Maybe add a comment?
Also, care to move the `when` under the `name` ?
```yaml
- name: foo
  when: [conditions]
  ansible.builtin....
```

Reply:
done